pkg/exporter/metrics.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package exporter
+
+import (
+	"io"
+
+	"github.com/cilium/tetragon/pkg/metrics/consts"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	eventsExportedTotal = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: consts.MetricsNamespace,
+		Name:      "events_exported_total",
+		Help:      "Total number of events exported",
+	})
+
+	eventsExportedBytesTotal = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: consts.MetricsNamespace,
+		Name:      "events_exported_bytes_total",
+		Help:      "Number of bytes exported for events",
+	})
+
+	eventsExportTimestamp = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: consts.MetricsNamespace,
+		Name:      "events_last_exported_timestamp",
+		Help:      "Timestamp of the most recent event to be exported",
+	})
+)
```

Comment:
what's the use case for this one? need to check if the export is lagging?

Reply:
Exactly.